MakerDAOs-Bridge-Invariant-Bot/src/agent.ts

```diff
@@ -0,0 +1,40 @@
+import { BlockEvent, ethers, Finding, getEthersProvider, HandleBlock } from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+import provideHandleBlockL1 from "./l1agentHandler";
+import provideHandleBlockL2 from "./l2agentHandler";
+import { NetworkData } from "./sharedConstant";
+
+const data: Record<number, NetworkData> = {
+  1: {
+    blockData: provideHandleBlockL1.handleBlock,
+  },
+  42161: {
+    blockData: provideHandleBlockL2.handleBlock,
+  },
+  10: {
+    blockData: provideHandleBlockL2.handleBlock,
+  },
+};
+const thisNetworkManager = new NetworkManager(data);
+export const asyncInitNetworkManager = (
```

Comment:
```suggestion
export const provideInitialize = (
```

Minor suggestion to change to the names the team typically uses